pytissueoptics/tests/testsVectors.py

```diff
@@ -0,0 +1,637 @@
+import envtest # modifies path
+from pytissueoptics import *
+import numpy as np
+import random
+
+inf = float("+inf")
+
+class TestVectors(envtest.PyTissueTestCase):
```

Comment:
They fail because they are not tested (I made all the non-tested methods fail by default). I have no idea if they work. I wanted to talk to you about the tests because I don't get what it does, So I can't really test without understanding. The plan is to test them.